apps/website/package.json

```diff
@@ -12,7 +12,7 @@
     "serve": "docusaurus serve",
     "write-translations": "docusaurus write-translations",
     "write-heading-ids": "docusaurus write-heading-ids",
-    "typecheck": "tsc"
+    "typecheck-bak": "tsc --noEmit"
```

Comment:
These types were not getting checked and there are quite a few errors. I renamed to `-bak` for now so it does not get run by turbo automatically. Will rename back to `typecheck` once the issues are fixed.

Reply:
The website doesn't compile? That's strange - I thought docusaurus would validate types

Reply:
It will build but fails type checking. I plan on fixing the issues in a subsequent PR.